src/spend/secret.rs

```diff
@@ -64,15 +64,15 @@ impl SecretKey {
 
     /// Derive the secret to deterministically construct a [`PublicKey`]
     pub fn public_key(&self) -> PublicKey {
-        let A = GENERATOR_EXTENDED * &self.a;
-        let B = GENERATOR_EXTENDED * &self.b;
+        let A = GENERATOR_EXTENDED * self.a;
```

Comment:
Shouldn't we `allow_non_snake_case`?

Reply:
It's enabled for the whole repo in `lib.rs`